Pre release check for RC (#2719)
googleforgames/agones: docs/governance/templates/release_issue.md

- [ ] Do a `helm install --namespace=agones-system agones agones/agones` 
    (`helm install --namespace=agones-system agones agones/agones --version={version}-rc` if release candidate) and a smoke test to confirm everything is working.
- [ ] Attach all assets found in the `release` folder to the draft Github Release.

> - [ ] If release candidate check the pre-release box end of the draft Github Release

Reviewer: ```suggestion
- [ ] If release candidate check the pre-release box on the draft Github Release
```
